cmd/controller/app/controller.go

```diff
@@ -222,13 +222,13 @@ func Run(rootCtx context.Context, opts *config.ControllerConfiguration) error {
 
 		// only run a controller if it's been enabled
 		if !enabledControllers.Has(n) {
-			log.V(logf.InfoLevel).Info("not starting controller as it's disabled")
+			log.V(logf.InfoLevel).Info("skipping disabled controller")
 			continue
 		}
 
 		// don't run clusterissuers controller if scoped to a single namespace
 		if ctx.Namespace != "" && n == clusterissuers.ControllerName {
-			log.V(logf.InfoLevel).Info("not starting controller as cert-manager has been scoped to a single namespace")
+			log.V(logf.InfoLevel).Info("skipping as cert-manager is scoped to a single namespace")
```

Comment:
suggestion: I think this is a better message but I worry that without "controller" in there it could be a little confusing. I'd vote for the following:

```suggestion
			log.V(logf.InfoLevel).Info("skipping controller as cert-manager is scoped to a single namespace")
```

Reply:
Trying to up skill & understand the context in which this log is used.

Is this log message used across all controllers? or just `cert-manager.controller`?
With the log Josh pasted at the top, there is already a key for `logger="cert-manager.controller"`.

Just wanting the check if with the `logger`, that's enough context?

If it's only used for one very specific controller, cert-manager, then perhaps the message should be:

```
log.V(logf.InfoLevel).Info("skipping cert-manager controller as it is scoped to a single namespace")
```